src/basic_recipes/tricontourf.jl

```diff
@@ -90,8 +117,14 @@ function compute_highcolor(eh, cmap)
     end
 end
 
-function Makie.plot!(c::Tricontourf{<:Tuple{<:AbstractVector{<:Real},<:AbstractVector{<:Real},<:AbstractVector{<:Real}}})
-    xs, ys, zs = c[1:3]
+function Makie.plot!(c::Tricontourf{<:Tuple{<:DelTri.Triangulation, <:AbstractVector{<:Real}}})
+    tri, zs = c[1:2]
+    xs = lift(tri) do tri 
+        return [DelTri.getx(p) for p in DelTri.each_point(tri)]
+    end 
+    ys = lift(tri) do tri 
```

Comment:
making `xs` and `ys` dependent on `tri` will cause triple updates down the line in the `onany` which depends on all three.